### PR TITLE
Azure Event Hubs "target" integration example and guidance

### PR DESCRIPTION
**Pull request by kaizimmerm**
Shows how to:
* Setup Azure Events Hub.
* Create a Ditto connection to forward messages and events into it.
* Small Spring Boot based sample to consume the events from an Azure Events Hub using its Kafka API.

Let me know what you think or even better give it a try 😄

Kai

### CODE REVIEW

**Comment by ghandim** [comment]
@kaizimmerm I will give it a try on my Azure environment

**Comment by kaizimmerm** [comment]
> Hi Kai.
> Just had a first quick look.
> Could you please add the Azure examples also to the root README.md like the other examples?

done.

**Comment by kartben** [comment]
It would probably be nice to have an ARM template for provisioning the Event Hub in just a few clicks?

**Comment by ghandim** [comment]
@kaizimmerm I would suggest that you should document that the event hub namespace is globally unique, or? Otherwise if you only copy and paste the create command for the namespace you get a weird response while creating the event hub namespace (as a Azure newbie ;-)).

File: azure/azure-eventhub-consumer/pom.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
[the remark this answers is not in the document]
Comment (kaizimmerm) [reply azure/azure-eventhub-consumer/pom.xml]: done

**Comment by kaizimmerm** [comment]
@ghandim done

**Comment by kaizimmerm** [comment]
@kartben in this case I think it will not make much of a difference as it is only a sample. However, I did use ARM for this guide https://github.com/eclipse/ditto/tree/master/deployment/azure

**Comment by ghandim** [comment]
@kaizimmerm Can confirm that the example is working as intended (after fumbling around with our corporate proxy and running into https://github.com/eclipse/ditto/pull/367 ;-))

File: azure/azure-eventhub-consumer/pom.xml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      <version>2.1.3.RELEASE</version>
+   </parent>
+   <groupId>org.eclipse.ditto.examples.azure</groupId>
+   <artifactId>azure-eventhub-consumer</artifactId>
+   <version>0.0.1-SNAPSHOT</version>
+   <name>azure-eventhub-consumer</name>
+   <description>Example application that consumes Events from Eclipse Ditto by Azure Event Hub.</description>
+
+   <properties>
+      <java.version>11</java.version>
```
Comment (ghandim) [inline azure/azure-eventhub-consumer/pom.xml]: @kaizimmerm I can confirm that the example also works Java 8.

**Comment by kaizimmerm** [comment]
@ghandim yeah, sry about that. At the end the solution was kind of simple but it cost me nevertheless two days to figure this out...

I will downgrade to Java 8 as well. Good catch, I always use 11 nowadays but no reason to enforce it.

File: azure/azure-eventhub-consumer/pom.xml (hunk 2 above)
Comment (kaizimmerm) [reply azure/azure-eventhub-consumer/pom.xml]: done

**Pull request merged**